### PR TITLE
Added tool and stats for naqfc_scripts

### PR DESCRIPTION
**Pull request by drnimbusrain**
- monet/util/tools.py
Added relative humidity tool to support new meteorological AIRNOW verification.

-  monet/util/mystats.py
Added stats to support new meteorological AIRNOW verification.

### CODE REVIEW

**Comment by bbakernoaa** [comment]
@drnimbusrain what is the difference between some of the functions?  NMB_TEMP and NMB?

if they are both NMB just with a different name we shouldn't duplicate them.

**Comment by drnimbusrain** [comment]
The NMB and NMB_TEMP are not the same. The NMB_TEMP uses the absolute value
of mean obs in the denominator, to account for possible negative average
temperature values (degrees C) in the winter.

Thank you,

Patrick

**Comment by bbakernoaa** [comment]
Then I suggest that we name it to reflect that difference rather than
specifically for TEMP.  This may be needed for other values or just convert
to kelvin and never worry about it.

**Comment by drnimbusrain** [comment]
OK we can switch the name (maybe NMB_ABS), but I was following your
different names for wind direction, and  I cannot think of any other
met/chem variable that would have negative values anyway.

Also, use of Kelvin is not appropriate for some statistical calculations
(e.g., such as NMB as you would expect artificially low values), so we
should rather always use degrees C for temperature.

Thank you,

Patrick

**Comment by bbakernoaa** [comment]
@drnimbusrain the naming in mystats is originally from Barron Henderson's PseudonetCDF.  I didn't change the names but do not think that we should name them specifically for an atmospheric variable.  MONET is becoming more generalized since its first inception when mystats was created.  I'm not even fond of the name mystats.

**Comment by drnimbusrain** [comment]
Fair enough, I'll change the NMB_TEMP to NMB_ABS and NME_TEMP_m to
NME_ABS_m.

File: monet/util/mystats.py [hunk 1]
```diff
@@ -386,12 +418,34 @@ def IOA(obs, mod, axis=None):
...
     return b
 
 
+def WDIOA_m(obs, mod, axis=None):
+    """ Wind Direction Index of Agreement, IOA (avoid single block error in np.ma)"""
+    obsmean = obs.mean(axis=axis)
+    if not axis is None:
+        obsmean = np.expand_dims(obsmean, axis=axis)
+    b = circlebias_m(mod - obs)
+
```
[the remark this answers is not in the document]
Comment (bbakernoaa) [reply monet/util/mystats.py]: These need to be changed to be if axis is not None.

[the remark this answers is not in the document]
Comment (bbakernoaa) [reply monet/util/mystats.py]: These need to be changed to be if axis is not None.

**Pull request closed without merging**